component/attr_cache/attr_cache.go

```diff
@@ -164,50 +163,56 @@ func (ac *AttrCache) OnConfigChange() {
 }
 
 // Helper Methods
-// deleteDirectory: Marks a directory and all its contents deleted.
-// This should only be called when ac.cacheDirs is false.
-// This marks items deleted instead of invalidating them.
-// That way if a request came in for a deleted item, we can respond from the cache.
-func (ac *AttrCache) deleteDirectory(path string, time time.Time) {
-	//get attrCacheItem
-	toBeDeleted, getErr := ac.cacheMap.get(path)
-	if getErr != nil || !toBeDeleted.exists() {
+
+// Record that the directory and all its contents have been deleted from the file system.
+// This should only be called when cacheDirs is false.
+// If cacheDirs is true, use deleteCachedDirectory instead.
+func (ac *AttrCache) deleteDirectory(path string, deletedAt time.Time) {
+	// get the entry to be marked deleted
+	value, found := ac.cache.get(path)
+	if !found {
```

Comment:
just calling it ```found``` is very elegant

Reply:
Thank you!
This was inspired by Go's map:
`value, ok := mapVar["key"]`